Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs

```diff
@@ -242,10 +243,10 @@ public enum ClearScreenOptions
 
                 break;
             case "SS3":
-                key = GetConsoleKey (terminator [0], values [0], ref mod);
+                key = GetConsoleKey (terminator [0], values [0], ref mod, ref keyChar);
```

Comment:
I don't love this.

Once there start being multiple outs or non-readonly refs, it's usually time to either use an implicit ValueTuple or a formal return type. Considering the scope of these, this is actually potentially a place where a ref struct could be quite beneficial.

Not worth upending that right now though. Just a note.